__tests__/app.test.js

```diff
@@ -140,3 +140,60 @@ describe("GET /api/articles", () => {
       });
   });
 });
+
+describe("GET /api/articles/:article_id/comments", () => {
+  test("should respond with 200 status code", () => {
+    return request(app).get("/api/articles/1/comments").expect(200);
```

Comment:
There is no need to make a separate test for a 200 status code as you have written in every other test that you're expecting the appropriate status code. This is a test for the sake of a green tick and isn't needed.